drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-backend/src/test/java/org/drools/workbench/screens/dtablexls/backend/server/DecisionTableXLSServiceImplCDITest.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.drools.workbench.screens.dtablexls.backend.server;
+
+import java.net.URL;
+
+import org.drools.workbench.models.guided.dtable.shared.conversion.ConversionResult;
+import org.drools.workbench.screens.dtablexls.service.DecisionTableXLSService;
+import org.guvnor.test.CDITestSetup;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.kie.workbench.common.services.shared.preferences.ApplicationPreferences;
+import org.uberfire.backend.server.util.Paths;
+import org.uberfire.backend.vfs.Path;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class DecisionTableXLSServiceImplCDITest extends CDITestSetup {
+
+    private ConversionResult conversionResult;
```

Comment:
Any reason for having this as a member variable? Why not just return from `convertResource()` directly? 
Stateless code is better code.

Reply:
Will do.